main.go

```diff
@@ -168,9 +168,14 @@ If the passed version does not exist, an error is signaled.`)
 - "autoalias": Apply auto aliasing to the provider.`)
 
 	cmd.PersistentFlags().StringVar(&context.UpstreamProviderName, "upstream-provider-name", "",
-		`The name of the upstream provider. 
+		`The name of the upstream provider.
 Required unless running from provider root and set in upgrade-config.yml.`)
 
+	cmd.PersistentFlags().BoolVar(&context.RemovePlugins, "remove-plugins", false,
```

Comment:
this means a corresponding update in the upgrade-provider-action, and ci-mgmt, yes?

Reply:
Yup, add `remove-plugins: true` to ci-mgmt for our providers' config file, and an input to the action